pkg/runtime/runtime.go

```diff
@@ -275,14 +261,8 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	}
 
 	a.flushOutstandingComponents()
-
 	a.initDirectMessaging(a.nameResolver)
```

Comment:
@jigargandhi initDirectMessaging uses the appchannel that is created. But now the appchannel creation is moved to [after direct messaging](https://github.com/dapr/dapr/pull/2215/files#diff-e56f9bb947923c6f8bb4c995bf4b9469a0ed117276223d16b177dcb04399a4bcR299) init. Can you change that?